content/applications/general/voip/devices_integrations.rst

```diff
@@ -0,0 +1,185 @@
+.. image:: devices_integrations/provider-zoiper-odoo.png
+   :align: center
+   :alt: Zoiper account setup, choosing the provider.
+
+Then, under :abbr:`SIP (Session Initiation Protocol)` options, enter the :guilabel:`Account name`,
+:guilabel:`Domain`, :guilabel:`Username`, and :guilabel:`Password`. All this information varies
+based on the account.
+
+.. tip::
+   To access this information via the *Axivox* portal, navigate to :menuselection:`Users --> Choose
+   user --> Edit --> SIP Identifiers tab`. The :guilabel:`SIP username`, :guilabel:`Domain`,
+   :guilabel:`SIP password`, and :guilabel:`Address of the proxy server` are all present in this
+   tab.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Zoiper Field
+     - Axivox Field
+   * - Account name
+     - *Can be anything*
+   * - Domain
+     - Domain
+   * - Username
+     - SIP username
+   * - Password
+     - SIP password
+
+Once this account information is entered in click the green :guilabel:`Register` button at the top
+of the screen. Once the registration information is checked Zoiper populates a message stating
+:guilabel:`Registration Status: OK`. Zoiper is now setup to make phone calls using the :abbr:`VoIP
+(Voice over Internet Protocol)` service.
+
+.. image:: devices_integrations/sip-options-zoiper.png
+   :align: center
+   :alt: Zoiper account setup, registration successful.
+
+Linphone
+========
+
+Linphone is an open source :abbr:`VoIP (Voice over Internet Protocol)` :abbr:`SIP (Session
+Initiation Protocol)` softphone used for voice, video, messaging (group and individual), as well as
+conference calls.
+
+To start using the Linphone app, first download it to the device via the `Linphone download page
+<https://new.linphone.org/technical-corner/linphone?qt-technical_corner=2#qt-technical_corner>`_.
+
+A mobile device is the most common installation, and this document covers setup on the Linphone IOS
+application. Screenshots and steps may differ depending on the setup.
+
+To begin configuring Linphone for use with a :abbr:`SIP (Session Initiation Protocol)` provider
+first open Linphone and a assistant screen appears. From this screen select :guilabel:`Use SIP
+Account`. Then on the following screen enter the :guilabel:`username`, :guilabel:`password`,
+:guilabel:`domain`, and :guilabel:`display name`. Then press :guilabel:`Login`. Linphone is ready to
+start making calls once there is a green button at the top of the application screen that reads
+:guilabel:`Connected`.
+
+.. image:: devices_integrations/linphone-odoo-setup.png
+   :align: center
+   :alt: Linphone account setup, registration successful.
+
+.. tip::
+   Linphone makes a variety of applications for mobile and desktop devices in operating systems,
+   such as Windows, Linux, Apple, and Android. Because Linphone is an open source project, many new
+   updates are released on a regular basis. See `Linphone's wiki-documentation page
+   <https://wiki.linphone.org/xwiki/wiki/public/view/Linphone/>`_.
```

Comment:
```suggestion
*Linphone* is an open-source :abbr:`VoIP (Voice over Internet Protocol)` :abbr:`SIP (Session
Initiation Protocol)` softphone, used for voice, video, messaging (group and individual), as well as
conference calls.

To start using the *Linphone* app, download it to the device, via the `Linphone download page
<https://new.linphone.org/technical-corner/linphone?qt-technical_corner=2#qt-technical_corner>`_.

A mobile device is the most common installation, and this document covers how to set up the
*Linphone* IOS application. Screenshots and steps may differ depending on the circumstances.

To begin configuring *Linphone* for use with a :abbr:`SIP (Session Initiation Protocol)` provider,
first open *Linphone*, and an assistant screen appears.

From this screen, select :guilabel:`Use SIP Account`. Then, on the following screen, enter the
:guilabel:`username`, :guilabel:`password`, :guilabel:`domain`, and :guilabel:`display name`. Once
complete, press :guilabel:`Login`. 

At this point, *Linphone* is ready to start making calls, once there is a green button at the top of
the application screen that reads, :guilabel:`Connected`.

.. image:: devices_integrations/linphone-odoo-setup.png
   :align: center
   :alt: Linphone account setup, registration successful.

.. tip::
   *Linphone* makes a variety of applications for mobile and desktop devices in operating systems,
   such as Windows, Linux, Apple, and Android. Because *Linphone* is an open-source project, many
   new updates are released on a regular basis.
   
   See `Linphone's wiki-documentation page
   <https://wiki.linphone.org/xwiki/wiki/public/view/Linphone/>`_.
```